src/pages/manage-my-registrations/update/[id].tsx

```diff
@@ -67,7 +74,14 @@ const RegistrationSummaryPage: FunctionComponent<RegistrationSummaryPageProps> =
                   actions={[
                     {
                       text: "Change",
-                      href: UpdatePageURLs.beaconDetails + registration.id,
+                      href:
+                        UpdatePageURLs.beaconDetails +
+                        registration.id +
+                        queryParams({
+                          review:
+                            UpdatePageURLs.registrationSummary +
+                            registration.id,
+                        }),
```

Comment:
On the page this links to, setting the `review` param to `URL` means "Continue button should submit then return to `URL`"